Docs/Help/New-BicepParameterFile.md

```diff
@@ -130,6 +138,21 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -AsHashtable
+The -AsHashtable prints all output as a hashtable instead of writing it to a file.
```

Comment:
Remove `The` in the start of the line or change it to:
`The -AsHashtable switch prints all output as a hashtable instead of writing it to a file.`

Reply:
😃 just re-used the description from Build-Bicep -AsHashtable.
I'll update.